### DISCUSSION TITLE
Dark Theme

### DISCUSSION THREAD

**Discussion by hardglitch** [kmaki565/EventLook#42] [Ideas]
Please, add Dark theme to the App! 
My eyes are burned

**Comment by kmaki565** [comment]
Thanks for the comment. According to my quick research, there is no straightforward way to accomplish this in WPF. 
It seems some UI frameworks provide a dark mode feature. I tried https://github.com/Kinnara/ModernWpf but it looks to me far from simple to support in my application. 

![image](https://github.com/kmaki565/EventLook/assets/16055659/3e6d395e-2dd2-43ad-825c-e82e6789292d)

**Reply by hardglitch** [reply]
What a pity! 
I hope you succeed.

**Comment by meowcatluke** [comment]
yes it needs a dark theme and an alerts section to run an app or email or trigger on alerts.

**Comment by umitseyhan75** [comment]
This.

**Comment by beedell-roke** [comment]
> https://github.com/kmaki565/EventLook/discussions/42#discussion-5220569

@hardglitch, this should be filed as an issue.
